Tests/helper.py

```diff
@@ -206,6 +207,7 @@ def _test_leak(self, core):
         start_mem = self._get_mem_usage()
         for cycle in range(self.iterations):
             core()
+            gc.collect()
```

Comment:
Did you want to talk about why you added this?

Reply:
I accidentally left this in here while I was debugging. I'll remove it.

Reply:
Actually, I realized now why I added this: without it the leak tests are non-deterministic. I could pad the memory limit to counteract the fact that it may not have hit the gc limit, but forcing garbage collection after each iteration ensures that the test is deterministic.

Reply:
This line is no longer present.